### ISSUE TITLE
./prepare_labels_from_txt.sh: line 50:  8689 Segmentation fault

### ISSUE THREAD

**Issue by HandsomeDevilv112** [CSTR-Edinburgh/merlin#286]
So, I might have accidentally needed a reformat. Now I'm having difficulty generating labels, and getting error messages that I don't really "get" Ive tried this with both run_aligner, and prepare_labels_from_txt, and haven't gotten anywhere with any of it. It should be noted that I've tried this with running aligner.sh, and ./02_prepare_labels.sh and have received the same error message. Festival is installed in /home/al/merlin/tools/festival, but if I try to change the FESTDIR to it directly, it cannot find it. I'm open to any and all assistance, So at your leisure, please help me out with some eli5.

./run_aligner.sh config.cfg
Step 1: creating label files from text...
creating a scheme file from text directory
generating utts from scheme file
UniSyn: using default diphone ax-ax for w-b
UniSyn: using default diphone ax-ax for hh-m
./prepare_labels_from_txt.sh: line 50:  8689 Segmentation fault      (core dumped) ${FESTDIR}/bin/festival -b ${WorkDir}/cmuarctic.scm
converting festival utts to labels...
c10x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c11x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c11x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c12x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c12x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c13x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c14x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c15x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c16x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c17x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c18x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c18x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c19x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c1x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c1x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c20x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c23x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c24x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c24x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c25x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c25x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c26x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c26x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
c27x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/tmp' for reading (No such file or directory)
normalizing label files for merlin...
c10x0.lab
c11x1.lab
c11x2.lab
c12x1.lab
c12x2.lab
c13x0.lab
c14x0.lab
c15x0.lab
c16x0.lab
c17x0.lab
c18x1.lab
c18x2.lab
c19x0.lab
c1x1.lab
c1x2.lab
c20x0.lab
c23x1.lab
c24x1.lab
c24x2.lab
c25x1.lab
c25x2.lab
c26x1.lab
c26x2.lab
c27x0.lab
c28x1.lab
Traceback (most recent call last):
  File "/home/al/merlin/misc/scripts/frontend/utils/normalize_lab_for_merlin.py", line 134, in <module>
    normalize_label_files(in_lab_file, out_lab_file, label_style, write_time_stamps)
  File "/home/al/merlin/misc/scripts/frontend/utils/normalize_lab_for_merlin.py", line 35, in normalize_label_files
    in_f = open(in_lab_file,'r')
IOError: [Errno 2] No such file or directory: '/home/al/merlin/egs/build_your_own_voice/s1/prompt-lab/full/c28x1.lab'
Labels are ready in: /home/al/merlin/egs/build_your_own_voice/s1/label_no_align !!
Step 2: forced-alignment using HTK tools...
Traceback (most recent call last):
  File "forced_alignment.py", line 379, in <module>
    aligner.prepare_training(file_id_list_name, wav_dir, lab_dir, multiple_speaker)
TypeError: prepare_training() takes exactly 6 arguments (5 given)

**Comment by HandsomeDevilv112** [comment]
and here is the error message when I run ./02_prepare_labels.sh in build_your_own_voice. Something else that's strange, is I have a series of txt and wav files with file format w#c# that does not appear in the processing list. I'm trying to run state alignment, to note.

al@Teletran-1:~/merlin/egs/build_your_own_voice/s1$ ./02_prepare_labels.sh /home/al/merlin/egs/build_your_own_voice/s1/wav /home/al/merlin/egs/build_your_own_voice/s1/txt /home/al/merlin/egs/build_your_own_voice/s1/labels
Step 2: 
Preparing labels...
preparing full-contextual labels using Festival frontend...
creating a scheme file from text directory
generating utts from scheme file
UniSyn: using default diphone ax-ax for w-b
UniSyn: using default diphone ax-ax for hh-m
/home/al/merlin/egs/build_your_own_voice/s1/scripts/prepare_labels_from_txt.sh: line 53: 29717 Segmentation fault      (core dumped) ${FESTDIR}/bin/festival -b ${out_dir}/$scheme_file
converting festival utts to labels...
c10x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c11x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c11x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c12x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c12x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c13x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c14x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c15x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c16x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c17x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c18x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c18x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c19x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c1x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c1x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c20x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c23x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c24x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c24x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c25x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c25x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c26x1
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c26x2
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
c27x0
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
gawk: fatal: cannot open file `/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/tmp' for reading (No such file or directory)
normalizing label files for merlin...
c10x0.lab
c11x1.lab
c11x2.lab
c12x1.lab
c12x2.lab
c13x0.lab
c14x0.lab
c15x0.lab
c16x0.lab
c17x0.lab
c18x1.lab
c18x2.lab
c19x0.lab
c1x1.lab
c1x2.lab
c20x0.lab
c23x1.lab
c24x1.lab
c24x2.lab
c25x1.lab
c25x2.lab
c26x1.lab
c26x2.lab
c27x0.lab
c28x1.lab
Traceback (most recent call last):
  File "/home/al/merlin/misc/scripts/frontend/utils/normalize_lab_for_merlin.py", line 134, in <module>
    normalize_label_files(in_lab_file, out_lab_file, label_style, write_time_stamps)
  File "/home/al/merlin/misc/scripts/frontend/utils/normalize_lab_for_merlin.py", line 35, in normalize_label_files
    in_f = open(in_lab_file,'r')
IOError: [Errno 2] No such file or directory: '/home/al/merlin/egs/build_your_own_voice/s1/labels/prompt-lab/full/c28x1.lab'
forced-alignment using HTK tools...
Traceback (most recent call last):
  File "/home/al/merlin/misc/scripts/alignment/state_align/forced_alignment.py", line 379, in <module>
    aligner.prepare_training(file_id_list_name, wav_dir, lab_dir, multiple_speaker)
TypeError: prepare_training() takes exactly 6 arguments (5 given)
ls: cannot access '/home/al/merlin/egs/build_your_own_voice/s1/labels/label_state_align': No such file or directory
Force-alignment unsucessful!! Please check /home/al/merlin/misc/scripts/alignment/state_align/forced_alignment.py
Copying labels to duration and acoustic data directories...
cp: cannot stat '/home/al/merlin/egs/build_your_own_voice/s1/labels/label_state_align': No such file or directory
cp: cannot stat '/home/al/merlin/egs/build_your_own_voice/s1/labels/label_state_align': No such file or directory
ls: cannot access '/home/al/merlin/egs/build_your_own_voice/s1/labels/label_state_align': No such file or directory
ls: cannot access '/home/al/merlin/egs/build_your_own_voice/s1/labels/label_state_align': No such file or directory
done...!
al@Teletran-1:~/merlin/egs/build_your_own_voice/s1$

**Comment by HandsomeDevilv112** [comment]
c28x1.wav was fine, c28x1.txt was much, much larger than it should have been. After editing c28x1.txt, I'm able to advance past "UniSyn: using default diphone ax-ax for hh-m" 
The issue appears to have been localized between the chair and the keyboard. Closing.

**Comment by HandsomeDevilv112** [comment]
Just to note, if you're getting this error, you need to change line 2 of "dumpfeats", not "dumpfeats.sh"
/home/al/merlin/tools/festival/examples/dumpfeats: 2: exec: /afs/inf.ed.ac.uk/group/cstr/projects/phd/s1432486/work/Festival_Tom/festival/bin/festival: not found

**Comment by vmontazeri** [comment]
What do you exactly mean by last comment? I'm getting the same error but I dont understand how to solve it.
